amo2kinto/exporter.py

```diff
@@ -43,23 +43,39 @@ def build_version_range(root, item, app_id, app_ver=None):
                             maxVersion=tA['maxVersion'])
 
 
-def is_related_to(item, app_id):
-    """Return True if the item relates to the given app_id."""
+def is_related_to(item, app_id, app_ver=None):
+    """Return True if the item relates to the given app_id (and app_ver, if passed)."""
     if not item.get('versionRange'):
         return True
 
     for vR in item['versionRange']:
         if not vR.get('targetApplication'):
             return True
+        if get_related_targetApplication(vR, app_id, app_ver) is not None:
+            return True
+    return False
 
-        for tA in vR['targetApplication']:
-            if tA['guid'] == app_id:
-                return True
 
-    return False
+def get_related_targetApplication(vR, app_id, app_ver):
+    """Return the first matching target application in this version range.
+    Returns None if there are no target applications or no matching ones."""
+    if not vR.get('targetApplication'):
+        return None
+
+    for tA in vR['targetApplication']:
+        if tA['guid'] == app_id:
```

Comment:
There used to be a `or 'guid' not in targetApplication` in the previous code. Maybe you want to do `tA.get('guid')` instead (to avoid `KeyError` and obtain `None`)